src/style.css

```diff
@@ -0,0 +1,170 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic);
+
+/* #region RESET */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --main-text-color: #060b35;
+  --second-text-color: #616070;
+  --text-size_s: 10px;
+}
+
+body {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+  padding: 0;
+  margin: 0;
+}
+
+h2,
+p {
+  margin: 0;
+  line-height: 0;
+}
+
+/* #endregion */
+
+/* #region STARS */
```

Comment:
it would be better if you create separated css files for your BEM blocks, reset and import those files to style.scss